src/examples/zkprogram/program-with-chunking.ts

```diff
@@ -0,0 +1,35 @@
+import { Field, Cache, Gadgets, ZkProgram } from 'o1js';
+
+let MyProgram = ZkProgram({
+  chunks: 2,
```

Comment:
Is the expectation on developers that they will specify number of chunks explicitly like this?  How will we recommend choosing a size?  Is there any cost to selecting a chunk size of 2 for a proof that is only 5k rows?

Reply:
Ideally, it should be performed automatically when in compile time we could count the number of rows and choose the chunks accordingly. But because chunking is still not working until we solve the memory limitation issue, this PR was just aiming at pinpointing the minimal changes required. There is definitely a cost implied with chunking, both in terms of performance and proof length. So you don't want to have more chunks than you really need.